frontend/src/scenes/insights/InsightContainer.tsx

```diff
@@ -187,11 +187,7 @@ export function InsightContainer(): JSX.Element {
             </Card>
             {renderTable()}
 
-            {preflight?.is_clickhouse_enabled &&
-            activeView === InsightType.FUNNELS &&
-            featureFlags[FEATURE_FLAGS.CORRELATION_ANALYSIS] ? (
-                <FunnelCorrelation />
-            ) : null}
+            {preflight?.is_clickhouse_enabled && activeView === InsightType.FUNNELS && <FunnelCorrelation />}
```

Comment:
non-blocking

Feels like this should be using `correlationAnalysisAvailable`? Out of scope for this change though.